File: test/fixtures/output/javascript/fetch/jsonObj-multiline.js
fetch("http://mockbin.com/har", {
  "method": "POST",
  "headers": {
    "content-type": "application/json"
  },
  "body": "{\"foo\":\"bar\"}"
})
.then(response => {
  console.log(response);
})
.catch(err => {
  console.error(err);
});

File: src/targets/java/nethttp/fixtures/jsonObj-multiline.java
HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create("http://mockbin.com/har"))
    .header("content-type", "application/json")
    .method("POST", HttpRequest.BodyPublishers.ofString("{\n  \"foo\": \"bar\"\n}"))
    .build();
HttpResponse<String> response = HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString());
System.out.println(response.body());